src/config/chain.config.ts

```diff
@@ -78,7 +78,8 @@ const chainConfig: Record<number, ChainConfig> = {
     claimManagerAddress: EWC_CLAIM_MANAGER_ADDRESS,
     stakingPoolFactoryAddress: '', // This is not deployed on EWC
     credentialRevocationRegistryAddress: '', // This is not deployed on EWC
-    claimsRevocationRegistryAddress: '', // This is not deployed on EWC
+    claimsRevocationRegistryAddress:
```

Comment:
Is it possible to export it from did-reg? I think we will need this in other packages like passport-did-auth

Reply:
I have added this here just to test it initially, it would definitely come from `ew-credentials` - https://github.com/energywebfoundation/ew-credentials/blob/develop/packages/credential-governance/src/chain-constants.ts